helm/README.md

```diff
@@ -120,95 +120,182 @@ For Helm chart, you'll need to craft a `values.yaml`.
     </tr>
     <tr>
       <td>
-       BOLD_SERVICES_UNLOCK_KEY
+       customLocalePath
+      </td>
+      <td>
+       Custom locale file path for Localization.
+      </td>
+    </tr>
+    </table>
+<br/>
+
+## Environment variables for configuring Application Startup in backend
+
+The below Environment variables are optional. If not provided a manual Application Startup configuration is needed.
```

Comment:
The following environment variables are optional. If not provided, a manual Application Startup configuration is needed.

Reply:
concern addressed